std/random.d

```diff
@@ -1656,7 +1657,7 @@ version (AnyARC4Random)
 }
 else
 {
-    private ulong bootstrapSeed() @nogc nothrow
+    private ulong bootstrapSeed()() @nogc nothrow
```

Comment:
On my machine changing this function to a template was necessary to avoid a symbol not found error with `betterC`. This seems like a bug.

Reply:
Why would that be a bug? Everything that works in betterC must be templated, as druntime and Phobos aren't linked. So of course we could needlessly templatize most of druntime, but what would be the point? Static functionality that can be precompiled once for many compilation cycles does provide benefits.

Reply:
I misunderstood how betterC treats Phobos then. I thought it treated it like any library.